app/database/migrations/2018_07_03_155435_create_suggestion_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+/**
+ * This migration adds suggestion table to store suggestions of information of locations 
+ * generated by users.
+ */
+
+class CreateSuggestionTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('suggestion',function(Blueprint $table){
+            $table->increments('id');
+            $table->integer('user_id')->unsigned();
```

Comment:
Did you test this with dropping all database tables and running "php artisan migrate"?

I'm surprised the following inconsistencies didn't make "php artisan migrate" fail:
The user table has a string type for the "id" field.
The location table also has a string type for "id" field.

Reply:
Actually I referenced line 102 and line 103 from the file "create_initial_database_schema.php" in migrations.
Just wondering if that needs to be changed too.
By the way, i don't know why the field "id" in suggestion table is int(10) instead of string. I used the same code "$table->increments('id')" as those in user table and location table.